config/environments/qa.rb

```diff
@@ -11,6 +11,9 @@
   # Base directory where user contributed annotations will be stored
   config.user_annotations_location = "/data/contributed_annotations/"
 
+  # Base directory where api created collections will be stored
+  config.api_collections_location = "/data/collections/"
```

Comment:
Shouldn't this be #{Rails.root}/data/collections/api to avoid breaking existing behavior?